### ISSUE TITLE
Screen reader not reading suggestion elements

### ISSUE THREAD

**Issue by cjrorvik** [moroshko/react-autosuggest#520]
**Observed behaviour:** 
When trying to navigate with arrows in the suggestions list. The screen reader does not read any of the elements.

**Expected behaviour:** 
To read the elements when navigating with arrows in the autosuggestion list

**How to reproduce**
- Go to https://en-tur.no/
- Enable VoiceOver
- Write something in one of the autosuggestion input fields. (ex. Oslo)
- Navigate in the suggestion list with the arrow keys.

**Testet browsers**
Chrome 64
Safari 11

Nothing is read :/

Any help would be highly appreciated!

**Comment by Bazzer588** [comment]
I can see the same issue on

macOS Sierra 10.12.6
Safari 12.0

Arrow keys do not work when Voiceover is switched on

Basically the input component does not retrieve events for onKeyDown, Up, Press for the arrow keys, when Voiceover is on

I don't think there's any fix for this - it's Voiceover causing the problem

**Comment by rheydt** [comment]
I am able to access the suggestions with Safari + Voiceover by pressing Shift + Control + Down Arrow.
Not sure why this particular combo is necessary.
